### PR TITLE
Remove unused scripts/build/package

### PR DESCRIPTION
**Pull request by amznestebanpapp**
Signed-off-by: Esteban Papp <81431996+amznestebanpapp@users.noreply.github.com>

Closes Issue #5049

### CODE REVIEW

**Review by DMWeiss-divesIn — APPROVED** [review]

**Review by amzn-changml — APPROVED** [review]
LGTM. Thanks for the cleanup

File: scripts/util/util.py [hunk 1]
```diff
@@ -13,26 +13,11 @@
 import subprocess
 
 
-class LyBuildError(Exception):
```
Comment (lumberyard-employee-dm) [inline scripts/util/util.py:16]: You also changed the permissions for this file and the `scripts/util/glob_to_regex.py` to remove the execute bit.

Did you intend to do that?

Comment (amznestebanpapp) [reply scripts/util/util.py:16]: did not intend, but these scripts should be executed from `python.cmd` anyway, so better if they dont have the execute bit. 
they are all loaded from other py scripts, so they dont need it, right?

Comment (lumberyard-employee-dm) [reply scripts/util/util.py:16]: Yes the execute bit is not needed if the script is not invoked directly by the loader.
It just means that `./util.py` will not work, but `python.cmd util.py` will always work

**Review by lumberyard-employee-dm — APPROVED** [review]

**Comment by amznestebanpapp** [comment]
Blocked by #6053, will wait for that issue to go in before this one goes in

**Pull request merged**